src/main/java/io/phasetwo/keycloak/model/jpa/JpaWebhookProvider.java

```diff
@@ -119,9 +120,11 @@ public WebhookEventModel storeEvent(
       em.flush();
       WebhookEventModel event = new WebhookEventAdapter(session, realm, em, e);
       return event;
-    } catch (EntityExistsException eee) {
+    } catch (ModelDuplicateException eee) {
```

Comment:
I didn't see this get thrown here. A `ModelDuplicateException` was thrown in the commit of the Keycloak transaction, but the `EntityExistsException` would be thrown on a duplicate in flush.